recipe/meta.yaml

```diff
@@ -15,16 +17,21 @@ source:
       - fix_ppc64le.patch  # [ppc64le]
 
 build:
-  number: 1
-  string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+  number: 2
+  string: py{{ PY_VER_MAJOR }}{{ PY_VER_MINOR }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
   skip: true  # [win or cuda_compiler_version == "10.2"]
   track_features: {{ "[librealsense-cuda]" if cuda_enabled else "" }}
   run_exports:
     # See https://github.com/IntelRealSense/librealsense/pull/3449
     - {{ pin_subpackage(name, max_pin='x.x') }}
+    # - {{ PYTHON }} ./wrappers/python/setup.py install
```

Comment:
Is this comment necessary?

Reply:
No that was a test as an alternative way to build the pyrealsense2 (which doesn't work iirc). I've removed it